vips/vip-307/bscmainnet.ts

```diff
@@ -0,0 +1,185 @@
+import { parseUnits } from "ethers/lib/utils";
+
+import { NETWORK_ADDRESSES } from "../../src/networkAddresses";
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+const { bscmainnet } = NETWORK_ADDRESSES;
+
+export const DEV_WALLET = "0x48e9d2128321cbf75cd108321459865357c00f15";
+export const WBNB = "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c";
+export const USDC = "0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d";
+export const ETH = "0x2170Ed0880ac9A755fd29B2688956BD959F933F8";
+export const BTC = "0x7130d2A12B9BCbFAe4f2634d864A1Ee1Ce3Ead9c";
+export const vUSDC = "0xecA88125a5ADbe82614ffC12D0DB554E2e2867C8";
+export const TOKEN_REDEEMER = "0xC53ffda840B51068C64b2E052a5715043f634bcd";
+
+export const WBNB_AMOUNT_DEV_FUND = parseUnits("1346.029696578437702", 18); // $834,400, assuming 1 BNB = $619.60
```

Comment:
`1346.029696578437702 * 619.60` is $834,000, not $834,400

Reply:
```suggestion
export const WBNB_AMOUNT_DEV_FUND = parseUnits("1346.675274370561653", 18); // $834,400, assuming 1 BNB = $619.60
```